### DISCUSSION TITLE
My github page isn't updating a whole day later

### DISCUSSION THREAD

**Discussion by DevRenamon-Alder** [community/community#137872] [Programming Help] [answered]
### Body

I added a patreon link to my game's website (work in progress) but its not updating a whole day later...

my website is digimon-digital-web.com, my repo is https://github.com/DevRenamon-Alder/DevRenamon-Alder.github.io

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by jovanovic-djo** [comment]
Before everything, you can try so called hard reload, which is ctrl + F5, it sometimes resolves that kind of issue. Feel free to try it. Otherwise, I advise you to check for any errors in your html code, so, verify if your previous commits were good.

**Reply by DevRenamon-Alder** [reply]
theres no errors, ive tried everything. its been 3 days. i tried to force cancel in github cli, i tried to rerun all the workflows after cancelling them. the workflows are queued another day later and they wont cancel bc it claims theyre completed.

**Reply by jovanovic-djo** [reply, marked as answer]
Good start,

Something caught my eye, if I understood your problem correctly, your site can not load on Github pages, as i have investigated your repository, your domain is this:
![image](https://github.com/user-attachments/assets/e61496c8-7737-4ae4-8e84-86a87cd2edbb)

That is custom domain name, for an instance, here is example of my project domain:
![image](https://github.com/user-attachments/assets/312b27fd-1593-417f-a86f-1537325019d6)

Pay attention to the domain on the second picture, there is .github.io.com part which is default when someone deploys project to Github Pages.

For further assistance, i would advise you to look for some of the examples of Github Pages deployment procedures.
Feel free to check out if one of these videos are helpful:
https://www.youtube.com/watch?v=BT4WzyT2g8k&ab_channel=AKRITIGOSWAMI
https://www.youtube.com/watch?v=OltY8JIaP-4&ab_channel=KennyYipCoding

**Reply by DevRenamon-Alder** [reply]
Thank you for that, it was the www. i typed in which it seems like you implied was the case. i appreciate yr help
